app/modules/conversations/conversation/conversation_service.py

```
@@ -285,6 +287,18 @@ def _on_fetch_done(t: asyncio.Task) -> None:
 
             fetch_task.add_done_callback(_on_fetch_done)
 
+            # Ensure repo is registered and cloned in repo manager (skip in local/VSCode mode)
+            if not local_mode and self.repo_manager and conversation.project_ids:
+                project_id_str = str(conversation.project_ids[0])
+                # Fast path: register any existing local copy (5s timeout)
+                asyncio.create_task(
+                    self._ensure_repo_in_repo_manager(project_id_str, user_id)
+                )
```

Ruff (0.15.1)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)